app/constants.py

```diff
@@ -1,14 +1,12 @@
 import os
 
 YOMA_AGENT_URL = os.getenv("ACAPY_YOMA_AGENT_URL", "http://localhost:3021")
+YOMA_AGENT_API_KEY = os.getenv("ACAPY_YOMA_AGENT_API_KEY", "adminApiKey")
 
 ECOSYSTEM_AGENT_URL = os.getenv("ACAPY_ECOSYSTEM_AGENT_URL", "http://localhost:4021")
 ECOSYSTEM_AGENT_API_KEY = os.getenv("ACAPY_ECOSYSTEM_AGENT_API_KEY", "adminApiKey")
 
 MEMBER_AGENT_URL = os.getenv("ACAPY_MEMBER_AGENT_URL", "http://localhost:4021")
 MEMBER_AGENT_API_KEY = os.getenv("ACAPY_MEMBER_AGENT_API_KEY", "adminApiKey")
 
-LEDGER_URL = os.getenv("LEDGER_NETWORK_URL", "http://localhost:9000/register")
```

Comment:
I see that for the tests there now is LEDGER_REGISTRATION_URL but It doesn't seem available to to the normal code?

Reply:
Registration through this URL is only for development purposes and not possible on production ledgers. In production the DID needs to be written to the ledger by an endorser. In the PR I'm working on right now, I set it up so that the Yoma agent writes public dids to the ledger using it's endorser DID.

I also try to use this setup as much as possible for tests, as it reflects how we will be interacting with the ledger in a production environment

Reply:
Ah alright, thanks for clarifying. Makes sense